doc/sources/releases.rst

```diff
@@ -0,0 +1,23 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _Releases:
+
+############################################
+Release Notes
+############################################
+
+   The updates of Intel® Extension for Scikit-learn for each release can be found in `Release Notes for Intel® oneAPI Data Analytics Library <https://www.intel.com/content/www/us/en/developer/articles/release-notes/oneapi-dal-release-notes.html>`_.
```

Comment:
this is a bit messy as scikit-learn user would be forwarded to oneDAL release notes. 
From github users perspective i would direct users to https://github.com/intel/scikit-learn-intelex/releases - there are release tags there with release notes specific to scikit-learn there.

Reply:
agree

Reply:
Fixed in https://github.com/intel/scikit-learn-intelex/pull/1323/commits/18c610980943defa287851ff154401a7f6fc9911